config.libsonnet

```diff
@@ -3,6 +3,22 @@
     // Selectors are inserted between {} in Prometheus queries.
     cephExporterSelector: 'job="rook-ceph-mgr"',
 
+    // Number of Ceph Managers which are reporting metrics
+    cephMgrCount: 3,
+    // Number of Ceph Monitors
+    cephMonCount: 3,
+    // Number of Ceph OSDs
+    cephOsdCount: 3,
```

Comment:
How come OSD count is configurable? Ideally it should be based on no of disks selected/available in storage nodes. We should not expect end user to configure the no of OSDs.

Suppose storage nodes in total have 20 disks and I set this count as 100, will it make sense?